.gitignore

```diff
@@ -3,3 +3,28 @@ build/
 
 .vagrant/
 *.swp
+
```

Comment:
Can we use `.idea/*` or `**/.idea/*` instead of all these lines?
Also that would be nice as a separate patch to keep this one focus.